apisix/plugins/log-rotate.lua

```diff
@@ -265,6 +265,22 @@ local function rotate_file(files, now_time, max_kept)
     end
 end
 
+local function get_custom_logfile_name()
+    local custom_file_name = core.config.local_conf()
+
+    local new_error_log = core.table.try_read_attr(local_conf, "nginx_config", "error_log")
+    local new_access_log = core.table.try_read_attr(local_conf, "nginx_config", "http", "access_log")
+
+    if new_access_log then
+        DEFAULT_ACCESS_LOG_FILENAME = new_access_log
```

Comment:
Should we do it during the loading phase because it will not change until it is reloaded or restarted.
> BTW, it's not a good idea to change the constants.

Reply:
https://github.com/apache/apisix/blob/d369b75a0fce6954573c3e67d05c56a8a48ef943/apisix/plugins/log-rotate.lua#L50-L51

Reply:
So, check for custom access and log values over here and then set the value for `DEFAULT_ACCESS_LOG_FILENAME` and `DEFAULT_ERROR_LOG_FILENAME` accordingly?

Reply:
I think it's better to do this in `init` phase
https://github.com/apache/apisix/blob/master/apisix/plugins/log-rotate.lua#L326